### PR TITLE
JS API: Documented the new publication property.

### PR DESCRIPTION
**Pull request by felixhageloh**
It contains the group and publication slug

### CODE REVIEW

File: source/js.md [hunk 1]
```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
...
+The `api.publication` property exposes information of the current publication. It contains the following properties:
+
+| Property      | Type        | Description         |
+| ------------- |-------------| ------------------- |
+| slug          | String      | publication identifier |
+| groupSlug     | String      | unique group identifier  |
+
+The `slug` and `groupSlug` properties are the building blocks of publication URLs, which have the structure
+
+*https://view.publitas.com/groupSlug/slug/`*
```
Comment (gfsanchez) [inline source/js.md]: test

File: source/js.md [hunk 2]
```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
...
+```
+
+The `api.publication` property exposes information of the current publication. It contains the following properties:
+
+| Property      | Type        | Description         |
+| ------------- |-------------| ------------------- |
+| slug          | String      | publication identifier |
+| groupSlug     | String      | unique group identifier  |
+
+The `slug` and `groupSlug` properties are the building blocks of publication URLs, which have the structure
```
Comment (gfsanchez) [inline source/js.md]: The `slug` and `groupSlug` properties are the building blocks of publication URLs, which have the structure:

File: source/js.md (hunk 1 above)
Comment (gfsanchez) [inline source/js.md]: there is a  ` at the end of the link?

File: source/js.md [hunk 3]
```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
...
+
+*https://view.publitas.com/groupSlug/slug/`*
+
+They can be used to construct a unique identifier for the current publication (to be used for custom analytics tracking, for example).
```
Comment (gfsanchez) [inline source/js.md]: They can be used to construct a unique identifier for the current publication (e.g. to be used for custom analytics tracking).

File: source/js.md [hunk 4]
```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
...
+
+Note that several publications can have the same slug as long as they are from different groups. To ensure to get a unique identifier for your publication, use both, the group and the publication slug.
```
Comment (gfsanchez) [inline source/js.md]: To ensure to get a unique identifier for your publication, use both the group and the publication slug.

File: source/js.md [hunk 5]
```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
...
+
+<aside class="notice">
+If you have a custom domain configured, your URLs will have the structure
```
Comment (gfsanchez) [inline source/js.md]: If you have a custom domain configured your URLs will have the structure:

File: source/js.md [hunk 6]
```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
...
+
+*http(s)://your-custom.domain/slug*
+
+and using just the publication slug might be sufficient if you are doing tracking per domain.
```
Comment (gfsanchez) [inline source/js.md]: Using just the publication slug might be sufficient if you are doing tracking per domain.

File: source/js.md (hunk 5 above)
Comment (felixhageloh) [inline source/js.md]: Most comma changes are correct, but I think this one is different in english https://www.ego4u.com/en/cram-up/writing/comma?10

Comment (gfsanchez) [inline source/js.md]: Agree!

**Comment by aseyedmehdi** [comment]
API doc improvements are already closed right @felixhageloh ?

**Comment by suweller** [comment]
PT story is accepted, merging this in.

**Pull request merged**